src/System.Management.Automation/engine/regex.cs

```diff
@@ -86,15 +86,17 @@ internal string PatternConvertedToRegex
         /// </summary>
         /// <param name="pattern">The wildcard pattern to match.</param>
         /// <returns>The constructed WildcardPattern object.</returns>
-        /// <remarks> if wildCardType == None, the pattern does not have wild cards</remarks>
+        /// <remarks>If wildCardType == None, the pattern does not have wild cards.</remarks>
```

Comment:
Is this `remarks` still relevant? I cannot find `wildCardType` in this file. If not, we can just remove it.

Reply:
Fixed.